src/Transition/InputRulesFiles/Rules23-1-0-to-23-2-0.md

```diff
@@ -0,0 +1,47 @@
+Input Changes version 22.2.0 to 23.1.0
+=============
+
+This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc.
+This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
+
+# Object Change: Coil:Heating:Fuel
+
+Fields 1-8, 10 remain the same.
+Field 9 has a name change from "Parasitic Electric Load {W}" 
+                            to "On Cycle Parasitic Electric Load {W}"
+Field 11 has a name change from "Parasitic Fuel Load {W}" 
+                             to "Off Cycle Parasitic Fuel Load {W}"
+
+See [PR#9925](https://github.com/NREL/EnergyPlus/pull/9925)
+
+# Object Change: Coil:Heating:Gas:MultiStage
+
+Fields 1-6, 8-10, 12-13, 15-16, 18-19 remain the same.
+Field 7 has a name change from "Parasitic Gas Load {W}" 
+                            to "Off Cycle Parasitic Gas Load {W}"
+Field 11 has a name change from "Stage 1 Parasitic Electric Load {W}" 
+                             to "Stage 1 On Cycle Parasitic Electric Load {W}"
+Field 14 has a name change from "Stage 2 Parasitic Electric Load {W}" 
+                             to "Stage 2 On Cycle Parasitic Electric Load {W}"
+Field 17 has a name change from "Stage 3 Parasitic Electric Load {W}" 
+                             to "Stage 3 On Cycle Parasitic Electric Load {W}"
+Field 20 has a name change from "Stage 4 Parasitic Electric Load {W}" 
+                             to "Stage 4 On Cycle Parasitic Electric Load {W}"
+
+See [PR#9925](https://github.com/NREL/EnergyPlus/pull/9925)
+
+# Object Change: Coil:Heating:Desuperheater
+
+Fields 1-8 remain the same.
+Field 9 has a name change from "Parasitic Electric Load {W}" 
+                            to "On Cycle Parasitic Electric Load {W}"
+
+See [PR#9925](https://github.com/NREL/EnergyPlus/pull/9925)
+
+# Object Change: Boiler:HotWater
+
+Fields 1-14, 16-17 remain the same.
+Field 15 has a name change from "Parasitic Electric Load {W}" 
+                             to "On Cycle Parasitic Electric Load {W}"
+
```

Comment:
@shorowit shouldn't this transition rules section also mention the new *Off Cycle Parasitic Fuel Load* input field at the end of the `Boiler:HotWater` class introduced in [this commit](https://github.com/NREL/EnergyPlus/pull/9925/commits/fdca71da11f6786c2be556476d82ab96b3913b7e#diff-37e00bb15fd2f5530f4a85e5e0c1d62837b3d97c6e2596e96304d2f1a86f51d6)?

Reply:
@aaron-boranian Since the new field is not required, there is nothing to transition.

Reply:
@shorowit ahhhh, that makes sense. Thanks!